doc/UsersGuide/source/IdealizedTC.rst

```diff
@@ -4,21 +4,30 @@
 .. _idealized-tc:
 
 **************************************
-Idealized Tropical Cyclone Case
+Idealized, Regional Tropical Cyclone Case
 **************************************
 
 The idealized, regional tropical cyclone case is derived from the I-HAFS configuration (:cite:t:`Wang2024`) and is designed to support controlled studies of tropical cyclone dynamics and forecast development. This configuration removes real-world data assimilation and ocean coupling, focusing solely on atmospheric forecasts using idealized inputs.
 
 Initial and lateral boundary conditions (ICs/LBCs) are derived from a large-scale, idealized global FV3-based atmospheric forecast. The initial vortex is constructed using the Reed and Jablonowski (2011) method, introducing a weak, balanced storm into an environment favorable for rapid intensification. The preprocessing system generates fixed distributions of geography-related variables and constructs the ICs/LBCs from ``tcvitals`` and GRIB input files.
```

Comment:
@jdkublnick can you add this somewhere? "The case is configured to run at 4-km resolution, with 81 vertical levels. The forecast is initialized on 24 August 2019, and initial/lateral boundary conditions are provided for up to a five-day forecast duration. "